keyboards/jlw/vault35_katana/info.json

```
@@ -0,0 +1,297 @@
+{
+    "keyboard_name": "vault 35 katana",
+    "manufacturer": "jlw",
+    "url": "https://github.com/josh-l-wang/vault35-katana",
+    "maintainer": "jlw",
+    "usb": {
+        "vid": "0x1209",
+        "pid": "0xA455",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F072",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["B0", "A7", "A6", "A5", "A4", "A3", "B5", "B4", "B3", "A15", "A14"],
+        "rows": ["B11", "B10", "B2", "B1"]
+    },
+    "encoder": {
+      "rotary": [
+        {"pin_a": "A1", "pin_b": "A2", "resolution": 4}
```

```suggestion
        {"pin_a": "A1", "pin_b": "A2"}
```
`"resolution": 4` is the default, so there is no need to specify it here.